warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -319,6 +319,8 @@ public class QueryOptions implements OptionDescriber {
 
     // filter for any key type (fi, event, tf)
     protected EventDataQueryFilter evaluationFilter;
+    protected EventDataQueryFilter fiEvaluationFilter;
+    protected EventDataQueryFilter eventEvaluationFilter;
```

Comment:
I see we now have `evaluationFilter`, `fiEvaluationFilter` and `eventEvaluationFilter`, and the comment mentions that we're filtering for `fi`, `event` and `tf`. Does this mean that the `evaluationFilter` is only used for tf's or is it used in other cases as well? 

Questions to consider:
* Is the `evaluationFilter` still used for anything?
* Is this an opportunity to rename the `evaluationFilter` to better explain how it is now used now that the two other filters have been added?
*